src/components/StoreDeployBlock.js

```diff
@@ -52,24 +68,20 @@ class StoreDeployBlock extends Component {
         <OrangeButton
           variant="pill"
           onClick={this.onDeployClick}
-          disabled={this.props.deploying}
+          disabled={deploying}
         >
-          {this.props.deploying ? "Deploying…" : "Deploy"}
+          {deploying ? "Deploying…" : "Deploy"}
         </OrangeButton>
 
-        {this.props.deployedTx ? (
+        {deployedTx ? (
```

Comment:
let's use full name instead of shorter: tx => transaction =)

Reply:
Created issue https://github.com/OpenCerts/admin-website/issues/96 for this.